src/util/ProjectIO.as

```diff
@@ -276,13 +340,16 @@ public class ProjectIO {
 			imageDict[svgData] = svgRoot;
 			doneFunction();
 		}
-		if (imageDict[svgData] != null) return; // already loading or loaded
+		if (imageDict[svgData] != null) {
+			// already loading or loaded
+			doneFunction();
+			return;
```

Comment:
We *need* `doneFunction()` to be called on every path, otherwise `decodeAllImages` never calls `allImagesLoaded` and we end up in project-load limbo.